Source/WebCore/html/HTMLScriptElement.idl

```diff
@@ -20,14 +20,16 @@
 [
     Exposed=Window
 ] interface HTMLScriptElement : HTMLElement {
-    [CEReactions=NotNeeded] attribute DOMString text;
+    [CEReactions=NotNeeded] attribute ScriptString text;
+    [CEReactions=Needed] attribute [LegacyNullToEmptyString] ScriptString innerText;
+    [CEReactions=Needed] attribute ScriptString? textContent;
```

Comment:
Remove from this PR as it's more complicated than just changing the type. It needs to be (DOMString or TrustedScript)?